src/eda_server/api/project.py

```diff
@@ -30,15 +30,15 @@ async def project_by_name_exists_or_404(db: AsyncSession, project_name: str):
 
 
 @router.get(
-    "/api/projects",
-    response_model=List[schema.ProjectList],
+    "/api/projects/",
+    response_model=LimitOffsetPage[schema.ProjectList],
     operation_id="list_projects",
     tags=["projects"],
 )
 async def list_projects(db: AsyncSession = Depends(get_db_session)):
     query = sa.select(projects.c.id, projects.c.url, projects.c.name)
     result = await db.execute(query)
-    return result.all()
+    return paginate(result.all())
```

Comment:
This will load all projects from the database and then apply pagination to the result.